wandb/apis/public.py

```diff
@@ -4700,26 +4704,148 @@ def save(self):
         }
         """
         )
+        introspect_query = gql(
+            """
+            query ProbeServerAddAliasesInput {
+               AddAliasesInputInfoType: __type(name: "AddAliasesInput") {
+                   name
+                   inputFields {
+                       name
+                   }
+                }
+            }
+            """
+        )
+        res = self.client.execute(introspect_query)
+        valid = res.get("AddAliasesInputInfoType") or None
```

Comment:
nit: when using `get`, it returns `None` if such key does not exist, so the `or None` is not needed